check_email

```diff
@@ -117,8 +116,10 @@ class ImapConnection():
     # TODO: regex is very expensive
     # search an alternative solution
     def parse_received(raw_email_received):
-        email_server_date_as_unix = raw_email_received.splitlines()[-1].strip()
-        match = re.search("(.*.... ..:..:.. [+-]....).*", email_server_date_as_unix)
+        # email_server_date_as_unix = raw_email_received.splitlines()[-1].strip()
+        # match = re.search("(.*.... ..:..:.. [+-]....).*", email_server_date_as_unix)
+        email_server_date_as_unix = raw_email_received
+        match = re.search(r";\s(.*)", email_server_date_as_unix)
```

Comment:
We should write a test for that. What is the raw input here?

@pdorschner do you have some examples

Reply:
Done